website/project/model.py

```diff
@@ -2749,6 +2749,8 @@ def embargo_registration(self, user, end_date, for_existing_registration=False):
         )
         # Embargo record needs to be saved to ensure the forward reference Node->Embargo
         self.embargo = embargo
+        if self.is_public:
+            self.set_privacy('private', Auth(user))
```

Comment:
Okay. Are there any adverse affects to manually setting it public instead of calling Node#set_privacy ?

Also thoughts on adding a `log=True` default param to Node#set_privacy?